controllers/articles.controller.js

```diff
@@ -0,0 +1,10 @@
+const { selectArticleById } = require("../models/articles.model");
+
+exports.getArticles = (req, res, next) => {
+  const { article_id } = req.params;
```

Comment:
Great use of object destructuring here to get article_id from the request params.